packages/ui/react-ui-deck/src/components/Deck/Deck.tsx

```diff
@@ -63,8 +63,13 @@ const DeckRoot = forwardRef<HTMLDivElement, DeckRootProps>(
   },
 );
 
+const defaults = {
+  // Min width for TLDraw to show tools.
+  size: 42,
```

Comment:
fyi @thure

Reply:
Sure, arbitrary changes here ahead of #7024 are inconsequential.